backend/core/src/user-preferences/entities/user-preferences.entity.ts

```diff
@@ -21,10 +18,7 @@ export class UserPreferences {
   @Expose()
   sendSmsNotifications?: boolean
 
-  @ManyToMany(() => Listing, (listing) => listing.favoritedPreferences, { nullable: true })
-  @JoinTable()
+  @Column("jsonb", { default: [] })
```

Comment:
`@Column("text", { array: true })`

Reply:
And then likely add `@IsString({ groups: [ValidationsGroupsEnum.default], each: true })`